modules/ROOT/pages/about-portals.adoc

```diff
@@ -4,23 +4,37 @@ include::_attributes.adoc[]
 endif::[]
 :imagesdir: ../assets/images/
 
-Exchange enables organizations to create a customized external view of Exchange
-within an organization with specific logo, banner image, text, and a favicon for the browser tab.
+Exchange public portals are available on the internet to everyone and enable outside developers to access an organization's REST, SOAP, and HTTP APIs.
 
-Exchange public portals are available on the internet to everyone and are used to
-externalize an organization's APIs to outside developers. The Exchange portals replace what used to be known
-as a Developer Portal. With Exchange portals you can customize the look and content of the portal.
+Exchange portals replace what used to be known as a Developer Portal.
 
-Exchange public portals let an organization share REST, SOAP, and HTTP APIs.
+You can customize the look and content of the portal with a logo, banner image, text, and a favicon for the browser tab.
 
-Exchange Administrators can share an API asset to a public portal as described in the sections that follow.
+== Share an Asset or Update the Asset Description
 
-== Update the Description of an Asset in a Portal
+Exchange Administrators can share an API asset to a public portal.
 
 . Locate the asset in your organization's Exchange.
 . Edit the asset's description, and when done, publish the asset.
 . Return to the portal and click the asset. The description updates automatically after changing the description in the asset.
 
+== Users From Other Anypoint Organizations
+
+Users can view a public portal without logging in, and must log in to use features such as reviewing assets, downloading assets, creating a client application to request API access, and accessing an API.
+
+Users from other Anypoint Platform organizations can use your public portal to register client applications so they can access your APIs. Because they do not belong to your organization, they are external users of your organization's portal.
+
+An organization administrator can see only the basic profile of an external user, which contains only this information:
+
+* Name
+* Username
+* Email
+* Primary organization name
+
+The first time an external user logs in to your public portal, the portal requests permission for read-only access to this basic profile. When the user approves the request, they are added in the Anypoint Platform tab *Access Management* > *Users* > *External Users*. As your organization's administrator, you can enable or disable each external user on that tab. Disabled external users cannot sign in to your public portal.
```

Comment:
```suggestion
The first time someone attempts to log in to your public portal, the portal requests permission for read-only access to this basic profile. When the person trying to log in approves this request, their information is added to the *External Users* section of *Users* on the *Access Management* tab in Anypoint Platform. However, they cannot log in until you, as your organization's administrator, enable them as an external user on that tab. If you disable them, they cannot sign in to your public portal.
```
A little unclear here: does the admin have to actively disable someone added as an external user? Does this persons info persist on record as either enabled or disabled? or does it disappear if NOT enabled? Reword to clarify.